packages/ui-codemods/lib/__node_tests__/__testfixtures__/updateV10Breaking/colors.input.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable */
```

Comment:
This is no longer needed, I ignore the whole folder in the main `eslint.config.mjs`